tests/foreman/cli/test_contentviewfilter.py

```diff
@@ -334,42 +286,31 @@ def test_create_cvf_with_multiple_repo_with_docker(self):
         repositories affected (yum and docker)
 
         """
-        try:
-            docker_repository = make_repository({
-                u'content-type': u'docker',
-                u'docker-upstream-name': u'busybox',
-                u'product-id': self.product['id'],
-                u'url': DOCKER_REGISTRY_HUB,
-            })
-        except CLIFactoryError as err:
-            self.fail(err)
-
-        result = ContentView.add_repository({
+        docker_repository = make_repository({
+            u'content-type': u'docker',
+            u'docker-upstream-name': u'busybox',
+            u'product-id': self.product['id'],
+            u'url': DOCKER_REGISTRY_HUB,
+        })
+        ContentView.add_repository({
             u'id': self.content_view['id'],
             u'repository-id': docker_repository['id'],
         })
-        self.assertEqual(result.return_code, 0)
-
-        repos = '{0},{1}'.format(self.repo['id'], docker_repository['id'])
-
-        result = ContentView.filter_create({
+        repos = [self.repo['id'], docker_repository['id']]
```

Comment:
This change will pass `repos` as a `list` and not as a `str` (which was the case). Is this intentional?